### PR TITLE
Update QUDA version to 1.1.0

### PR DESCRIPTION
**Pull request by maddyscientist**
This should be final PR into release/1.1.x prior to us tagging and releasing 1.1.0.
* Update QUDA version to 1.1.0
* Update README.md and NEWS files for the release

Reviewers: please have a read through the changes on this PR.  Are we missing anything substantial that should be included in the README and NEWS updates?

### CODE REVIEW

**Review by cpviolator — APPROVED** [review]

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,84 @@
...
+  `QudaMultigridParam::use_mma` parameter.
+
+- Improved support of managed memory through the addition of a
+  prefetch API.  This can dramatically improve the performance of the
+  multigrid setup when oversubscribing the memory.
+
+- Improved the performance of using MILC RHMC with QUDA
+
+- Add support for a new internal data order FLOAT8.  This can be
+  enabled for applicable half precision field types (instead of
```
Comment (mathiaswagner) [inline NEWS]: when we make it default it should say 'disabled'

File: NEWS [hunk 2]
```diff
@@ -1,3 +1,84 @@
...
+  https://github.com/lattice/quda/wiki/Multi-GPU-with-NVSHMEM for more
+  details.
+
+- Addition of the MSPCG preconditioned CG solver for Möbius
+  fermions. See
+  https://github.com/lattice/quda/wiki/The-Multi-Splitting-Preconditioned-Conjugate-Gradient-(MSPCG),-an-application-of-the-additive-Schwarz-Method
+  for more details.
+
+- Addition of the Exact One Flavor Algorithm (EOFA) for Möbius
+  fermions.
```
Comment (hummingtree) [inline NEWS]:
```suggestion
  fermions. See
  https://github.com/lattice/quda/wiki/The-Exact-One-Flavor-Algorithm-(EOFA)
  for more details
```

File: NEWS [hunk 3]
```diff
@@ -1,3 +1,84 @@
...
+- Addition of a fully GPU native Implicitly Restarted Arnoldi
+  eigensolver (as opposed to partially relying on ARPACK).  See
+  https://github.com/lattice/quda/wiki/QUDA%27s-eigensolvers#implicitly-restarted-arnoldi-eigensolver
+  for more details.
+
+- Significantly reduced latency for reduction kernels through the use
+  of heterogeneous atomics.  Requires CUDA 11.0+.
+
+- Addition of support for a split-grid multi-RHS solver.  See
+  https://github.com/lattice/quda/wiki/Split-Gridfor more details.
```
Comment (hummingtree) [inline NEWS]:
```suggestion
- Addition of support for a split-grid multi-RHS solver. See
  https://github.com/lattice/quda/wiki/Split-Grid for more details.
```

File: NEWS [hunk 4]
```diff
@@ -1,3 +1,84 @@
+Version 1.1.0 - July 2021
+
+- Add support for NVSHMEM communication for the Dslash operators, for
+  significantly improved strong scaling.  See
```
Comment (hummingtree) [inline NEWS:4]:
```suggestion
  significantly improved strong scaling. See
```

File: NEWS [hunk 5]
```diff
@@ -1,3 +1,84 @@
...
+- Addition of support for a split-grid multi-RHS solver.  See
+  https://github.com/lattice/quda/wiki/Split-Gridfor more details.
+
+- Continued work on enhancing and refining the staggered multigrid
+  algorithm.  The MILC interface can now drive the staggered multigrid
+  solver.
+
+- Multigrid setup can now use tensor cores on Volta, Turing and Ampere
+  GPUs to accelerate the calculation.  Enable with the
+  `QudaMultigridParam::use_mma` parameter.
```
Comment (hummingtree) [inline NEWS:35]:
```suggestion
  GPUs to accelerate the calculation. Enable with the
  `QudaMultigridParam::use_mma` parameter.
```

File: NEWS (hunk 4 above)
Comment (maddyscientist) [reply NEWS:4]: Ha, if you ask me to change every double space to single space after a period, you're asking for a global edit 😉 

This is an age thing it seems:
> The use of single vs. double-spacing after periods has become a great generational debate, with everyone from Gen X on up falling predominantly onto Team Double Space, while Millennials and Gen Z fight on the side of the single space.

https://www.thesaurus.com/e/writing/is-there-1-space-or-2-after-a-period/

Comment (hummingtree) [reply NEWS:4]: Great to know that... I never know you can put two spaces after a period.

Comment (maddyscientist) [reply NEWS:4]: In other words: I am old and you are not 😜

Comment (hummingtree) [reply NEWS:4]: Haha ...

Comment (weinbe2) [reply NEWS:4]: Why waste the extra bytes on extra spaces? =)

Comment (mathiaswagner) [reply NEWS:4]: It makes me feel young :-)

**Review by weinbe2 — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -227,7 +236,15 @@ When taking advantage of block CG, please also cite:
 M. A. Clark, A. Strelchenko, A. Vaquero, M. Wagner, and E. Weinberg,
 "Pushing Memory Bandwidth Limitations Through Efficient
 Implementations of Block-Krylov Space Solvers on GPUs,"
-To be published in Comput. Phys. Commun. (2018) [arXiv:1710.09745 [hep-lat]].
+Comput. Phys. Commun. 233 (2018), 29-40 [arXiv:1710.09745 [hep-lat]].
+
+When taking advantage of the Möbius MSPC solver, please also cite:
```
Comment (hummingtree) [inline README.md]:
```suggestion
When taking advantage of the Möbius MSPCG solver, please also cite:
```

File: README.md [hunk 7]
```diff
@@ -227,7 +236,15 @@ When taking advantage of block CG, please also cite:
...
+
+Jiqun Tu, M. A. Clark, Chulwoo Jung, Robert Mawhinney, "Solving DWF
+Dirac Equation Using Multi-splitting Preconditioned Conjugate Gradient
+with Tensor Cores on NVIDIA GPUs,".  To be published in the Platform
+of Advanced Scientific Computing (PASC21) [arXiv:2104.05615
+[hep-lat]].
```
Comment (hummingtree) [inline README.md]:
```suggestion
with Tensor Cores on NVIDIA GPUs," published in the Platform of
Advanced Scientific Computing (PASC21) [arXiv:2104.05615[hep-lat]].
```

**Review by hummingtree — APPROVED** [review]

File: NEWS [hunk 8]
```diff
@@ -1,3 +1,93 @@
+Version 1.1.0 - July 2021
```
Comment (mathiaswagner) [inline NEWS]: JUly->October

File: README.md [hunk 9]
```diff
@@ -44,12 +44,12 @@ For more details we refer the user to the wiki:
 ## Software Compatibility:
 
 The library has been tested under Linux (CentOS 7 and Ubuntu 18.04)
-using releases 7.5 through 10.2 of the CUDA toolkit.  Earlier versions
+using releases 10.1 through 11.3 of the CUDA toolkit.  Earlier versions
```
Comment (mathiaswagner) [inline README.md]: 11.3 -> 11.4

**Review by mathiaswagner — DISMISSED** [review]

**Pull request merged**